src/HW_19_5.py

```diff
@@ -0,0 +1,25 @@
+import sqlite3
+
+# Поточний шлях до бази даних SQLite
+db_path = 'C:\\Users\\Юлія\\Desktop\\база даних для длмашки.sqlite'
+
+# Встановлення з'єднання з базою даних
+connection = sqlite3.connect(db_path)
+
+# Створення курсора
+cursor = connection.cursor()
+
+# Виконання запиту
+query = "SELECT age, COUNT(*) as user_count FROM users GROUP BY age HAVING COUNT(*) > 1 ORDER BY user_count DESC;"
```

Comment:
nit: aliases could be used inside` having` statement and it is better to write sql queries in multiple lines for better readability
```suggestion
query = "SELECT age, COUNT(*) as user_count FROM users GROUP BY age HAVING user_count > 1 ORDER BY user_count DESC;"
```